client/src/components/ImageApp.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+class ImageApp extends React.Component {
+	render() {
+		return (
+			<div id="image-app">
+        <img src="https://fug-images-paintings.sfo2.digitaloceanspaces.com/fugfam-big-poppy-bday.jpg" width="133" height="100" />
```

Comment:
am I the asshole or is the indentation the asshole? last time I'll mention it (probably a lie)